internal/log_analysis/log_processor/logschema/logschema.go

```
@@ -57,6 +59,13 @@ type Schema struct {
 	Fields       []FieldSchema           `json:"fields" yaml:"fields"`
 }
 
+func (s *Schema) Clone() *Schema {
+	data, _ := jsoniter.Marshal(s)
+	out := Schema{}
+	_ = jsoniter.Unmarshal(data, &out)
+	return &out
```

We shouldn't ignore the errors. This shouldn't happen though, so i think we could just panic.

---

Fixed